index.js

```diff
@@ -18,6 +18,10 @@ var fs = require('fs');
 var path = require('path');
 var builtins = require('browserify/lib/builtins');
 
+builtins.dns = require.resolve('runtime-node-dns');
+builtins.net = require.resolve('runtime-node-net');
+builtins.http = require.resolve('http-node');
```

Comment:
Yes, but this is the way `runtime-node-net` maps itself to `net` in it's tests. So, (unless there is a better way) I use this to map them.